src/components/bs5/navbar/navbar.hbs

```diff
@@ -31,64 +31,54 @@
                 <li class="nav-item dropdown">
                     <a class="nav-link dropdown-toggle" href="{{target_url}}" id="menuDropdown" role="button"
                         aria-controls="" data-bs-toggle="dropdown" aria-expanded="false" aria-selected="false"
-                        aria-label="Toggle navigation">{{title}}
+                        aria-label="Toggle navigation">
+                            {{title}}
                             <svg class="toggle_icon">
                                 <use href="assets/img/svg-icons.svg#qld__icon__chevron-down"></use>
```

Comment:
Hey @MitchAnderson93, can we make this path "assets/img/svg-icons.svg" a {{variable}} also?  We'll need to pass in a unique path in Squiz CMS.

Reply:
Done all are these now:
<use href="{{metadata.options.icon-root}}#qld__icon__close"></use>